migration/migrator/migrations/course/20210320080101_add_pause_timer.py

```diff
@@ -0,0 +1,13 @@
+from datetime import datetime
+native_utc_dt = datetime.utcnow();
+utc_string = native_utc_dt.strftime("%Y-%m-%d %H:%M:%S") + " UTC";
+
+def up(config, database, semester, course):
+    database.execute('ALTER TABLE queue ADD COLUMN IF NOT EXISTS time_paused INTEGER;')
+    database.execute('ALTER TABLE queue ADD COLUMN IF NOT EXISTS time_paused_start TIMESTAMP WITH TIME ZONE;')
+    database.execute('UPDATE queue SET time_paused = 0;')
+    database.execute('UPDATE queue SET time_paused_start = \'' + utc_string + '\' WHERE paused = true;')
+
+def down(config, database, semester, course):
+    database.execute('ALTER TABLE queue DROP COLUMN IF EXISTS time_paused;')
```

Comment:
Do we need to delete these columns on the down migration?  Why can't we leave them?

Reply:
My apologies, I thought this was standard for down migrations. There is no actual reason to delete them so I'll edit that out.